README.md

```diff
@@ -54,16 +55,27 @@ In this step, we'll view the same HSSD scene on the Quest VR headset. No HITL se
 5. Put on your headset. The app may already be running. If not, or if you encounter issues, hit the Quest home button and it should let you quit the `siro_hitl_vr_client` program. After you quit, you can usually find the icon for the program in your list of recently-run programs. You should re-launch it.
 6. When in the scene, press and hold forward on either VR controller thumbstick to aim for a teleportation destination, then release the thumbstick to teleport.
 
-# Testing the interactive Fetch task with the HITL server
-This is the full, interactive demo. You can use either the Unity Editor (VR emulation) or the Quest VR headset. You can run the Unity Editor on the same machine as the HITL server, side-by-side--this is shown in the screenshot above.
+See [these troubleshooting steps](#deployment-to-the-vr-headset) if you have issues deploying the client to your VR device.
+
+## Testing with the HITL server
+[This document](https://github.com/facebookresearch/habitat-lab/blob/main/examples/siro_sandbox/VR_HITL.md) contains complete information for setting up the server.
+
+### Installing external Habitat data
+The `Assets/Resources/data` folder is a subset of the Habitat `data/` folder. When the Habitat server indicates that a file must be loaded, the Unity client will look for an asset with the same name within its own data folder.
```

Comment:
We should probably still include instructions here for the file-based replay rendering test. Alternately, users should be encouraged to locally test at least one model built by the data pipeline. Otherwise, users are forced to troubleshoot both network problems and data-pipeline problems simultaneously (e.g., what if nothing shows up on-screen? Did the connection fail? Did the data processing fail?)